src/tables/cff.js

```diff
@@ -1167,14 +1167,15 @@ function glyphToOps(glyph) {
             const _23 = 2 / 3;
 
             // We're going to create a new command so we don't change the original path.
+            // Since every coordinates are relativ, we round() them ASAP to avoid propagating errors
```

Comment:
Could you correct the spelling? Something like:

```// Since all coordinates are relative, we round() them ASAP to avoid propagating errors.```

Reply:
Fixed :)